Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:18.7.0 as builder
+
+RUN mkdir -p /home/node/app
+WORKDIR /home/node/app
+COPY package*.json /home/node/app/
```

Comment:
this is not needed with * but we need yarn.lock additionally

Reply:
Great , will change them.

Reply:
thanks